ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -0,0 +1,89 @@
+SUPPORTED_NEXTCLADE_SEROTYPES = ['denv1', 'denv2', 'denv3', 'denv4']
+SEROTYPE_CONSTRAINTS = '|'.join(SUPPORTED_NEXTCLADE_SEROTYPES)
+
+rule nextclade_denvX:
+    """
+    For each type, classify into the appropriate subtype
+    """
+    input:
+        sequences="results/sequences_{serotype}.fasta",
+        dataset="../nextclade_data/{serotype}",
+    output:
+        nextclade_denvX="data/nextclade_results/nextclade_{serotype}.tsv",
+    threads: 4
+    params:
+        min_length=config["nextclade"]["min_length"],
+        min_seed_cover=config["nextclade"]["min_seed_cover"],
+    wildcard_constraints:
+        serotype=SEROTYPE_CONSTRAINTS
+    shell:
+        """
+        nextclade run \
+          --input-dataset {input.dataset} \
+          -j {threads} \
+          --output-tsv {output.nextclade_denvX} \
+          --min-length {params.min_length} \
+          --min-seed-cover {params.min_seed_cover} \
+          --silent \
+          {input.sequences}
+        """
+
+rule concat_nextclade_subtype_results:
+    """
+    Concatenate all the nextclade results for dengue subtype classification
+    """
+    input:
+        expand("data/nextclade_results/nextclade_{serotype}.tsv", serotype=SUPPORTED_NEXTCLADE_SEROTYPES),
+    output:
+        nextclade_subtypes="results/nextclade_subtypes.tsv",
+    params:
+        id_field=config["transform"]["id_field"],
+        nextclade_field=config["nextclade"]["nextclade_field"],
+    shell:
+        """
+        echo "{params.id_field},{params.nextclade_field}" \
+        | tr ',' '\t' \
+        > {output.nextclade_subtypes}
```

Comment:
This part of the `concat_nextclade_subtype_results:` does not seem to be parallized as expected. This restarts the "results/nextclade_subtypes.tsv" about 4 times, so the final output is only partially there from my test. Perhaps can do an aggregate, and then add the header as a final step.

Reply:
Nevermind, I missed the "expand" statement. This rule is called once and should work. I'll revert the fixup